indico/modules/events/registration/models/items.py

```diff
@@ -437,6 +437,10 @@ def view_data(self):
     def _get_default_log_data(self):
         return {'Section ID': self.id}
 
+    @make_interceptable
+    def is_visible(self, is_management):
```

Comment:
While testing i noticed that we already have an `is_visible` hybrid properties on regform items. So we have to find yet another name... :/

I can't think of a name I truly like, so let's maybe go for `is_visible_in_summary` for now.

Reply:
Updated.